### PR TITLE
docs: Update text widget documentation formatting and table styling

### LINKED ISSUE #11059: [docs]:  Update the documentation for the Text Component
### The entire URL of the documentation with the issue

https://docs.tooljet.com/docs/widgets/text

**Formatting updates:**
- Replace  "`fx`" with "**fx**" wherever applicable. ( i.e. remove the backticks (``) make it lowercase and bold -> **fx**)

**Content updates:**
- Devices 
  -  In the devices section, change from text content to a table - [reference](https://docs.tooljet.com/docs/widgets/text-input/#devices)
  -  Add a full stop at the end of the sentence in the "Description" column.
  
**The changes are to be implemented in two files -** 
 Next - ToolJet/docs/docs/widgets/text.md
 Version 2.50.0 (LTS) - ToolJet/docs/versioned_docs/version-2.50.0-LTS/widgets/text.md

### If the issue is confirmed, would you be willing to submit a pull request?

No

### Code of Conduct

- [X] I agree to follow the ToolJet Code of Conduct

### PR DESCRIPTION
**Pull request by OoONANCY**
Fixes #11059

This PR updates the Text widget documentation with the following changes:

1. Formatting Updates:
   - Replaced all instances of `fx` with **fx** to maintain consistent styling
   - Made the text lowercase and bold as per requirements

2. Content Structure Changes in Devices Section:
   - Converted text content to table format matching Additional Actions section styling
   - Added proper div styling for column headers:
     ```
     | <div style={{ width:"100px"}}> Action </div> | <div style={{ width:"150px"}}> Description </div> | <div style={{ width:"250px"}}> Configuration Options </div>|
     ```
   - Ensured all sentences in Description column end with a full stop

Files modified:
- docs/docs/widgets/text.md
- docs/versioned_docs/version-2.50.0-LTS/widgets/text.md

---
I have tested the changes locally and verified that the documentation renders correctly.

### CODE REVIEW

File: docs/docs/widgets/text.md [hunk 1]
```diff
@@ -54,20 +54,16 @@ Check the **component specific actions** available for this component **[here](/
...
-**Show on desktop**
-
-Makes the component visible in desktop view. You can set it with the toggle button or dynamically configure the value by clicking on `fx` and entering a logical expression.
-
-**Show on mobile**
-
-Makes the component visible in mobile view. You can set it with the toggle button or dynamically configure the value by clicking on `fx` and entering a logical expression.
+| <div style={{ width:"100px"}}> Action </div> | <div style={{ width:"150px"}}> Description </div> | <div style={{ width:"250px"}}> Configuration Options </div>|
+|------------|:-------------|:-----------------|
+| Show on desktop | Makes the component visible in desktop view. You can set it with the toggle button or dynamically configure the value by clicking on **fx** and entering a logical expression. |
```
Comment (RudraPratik30) [inline docs/docs/widgets/text.md:65]: Hey, can you please check this once again?
I guess there are only 2 columns in this row, while 3 columns were required.

File: docs/docs/widgets/text.md [hunk 2]
```diff
@@ -76,29 +72,29 @@ Makes the component visible in mobile view. You can set it with the toggle butto
...
-| Text Decoration   | Adds an underline, overline, line-through, or a combination of lines to selected text. | Select one of the available options - `none(default)`, `underline`, `overline`, and `strike-through` or dynamically configure it using `fx` |
-| Transformation | Dictates the capitalization of text. | Select one of the available options - `none (default)`, `uppercase`, `lowercase`, `capitalize` or dynamically configure it using `fx`|
-| Letter spacing | Determines the space between each letter. | Enter a number as the value (example: `15`) or dynamically configure it using `fx`|
-| Word spacing | Determines the space between each word. | Enter a number as the value (example: `15`) or dynamically configure it using `fx`|
-| Font variant | Adjusts the text appearance by applying font variations. | Select one of the available options - `normal`, `inherit`, `small-caps`, `initial` or dynamically configure it using `fx`|
+| Size         | Dimensions of the characters in a font. | Enter any number between `1-100` or dynamically configure it using **fx** |
```
Comment (RudraPratik30) [inline docs/docs/widgets/text.md:75]: can you please add full stops at the end of these sentences, also wherever applicable in the doc?

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey @OoONANCY can you please make the mentioned changes in both versions?

**Comment by PriteshKiri** [comment]
Hey @OoONANCY,

Any updates on the review comments? Please let me know if you are facing issues

**Comment by PriteshKiri** [comment]
Closing the PR due to lack of progress.

**Pull request closed without merging**